.github/workflows/nightly_tests_and_veracode.yml

```diff
@@ -36,7 +39,7 @@ jobs:
     name: "Pipeline SAST Scan"
     runs-on: ubuntu-latest
     if: >
-      (github.ref_name == 'master' || startsWith(github.ref_name, 'release/') || github.event_name == 'schedule') &&
+      (github.ref_name == 'master' || startsWith(github.ref_name, 'release/') || github.event_name == 'pull_request') &&

```

Comment:
this was just for testing right? should be reverted

you could add a workflow_dispatch trigger for the future (and add an additional `|| github.event_name == 'workflow_dispatch'` here)